tc-report/_includes/head.html

```diff
@@ -26,11 +26,11 @@
         });
     </script>
   <!-- Favicons -->
-  <link rel="apple-touch-icon" sizes="180x180" href="{{ '/apple-touch-icon.png' | prepend: site.baseurl }}">
-  <link rel="icon" type="image/png" sizes="32x32" href="{{ '/favicon-32x32.png' | prepend: site.baseurl }}">
-  <link rel="icon" type="image/png" sizes="16x16" href="{{ '/favicon-16x16.png' | prepend: site.baseurl }}">
-  <link rel="manifest" href="{{ '/site.webmanifest' | prepend: site.baseurl }}">
-  <link rel="mask-icon" href="{{ '/safari-pinned-tab.svg' | prepend: site.baseurl }}" color="#5bbad5">
+  <link rel="apple-touch-icon" sizes="180x180" href="{{ '/apple-touch-icon.png' | prepend: site.asset_url }}">
+  <link rel="icon" type="image/png" sizes="32x32" href="{{ '/favicon-32x32.png' | prepend: site.asset_url }}">
+  <link rel="icon" type="image/png" sizes="16x16" href="{{ '/favicon-16x16.png' | prepend: site.asset_url }}">
+  <link rel="manifest" href="{{ '/site.webmanifest' | prepend: site.asset_url }}">
```

Comment:
This change makes sense and points to the correct webmanifest (previously was pointing to [this one](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/apps/web/public/site.webmanifest)). However, the correct site.webmanifest is missing some information and has invalid file paths. You can test this in [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/application/manifests/index.html). This should do the trick for English (we have an [issue open](https://github.com/GCTC-NTGC/gc-digital-talent/issues/4780) for French and other locales):

```
{
    "name": "GC Digital Talent",
    "short_name": "GC Digital Talent",
    "icons": [
        {
            "src": "/tc-report/assets/android-chrome-192x192.png",
            "sizes": "192x192",
            "type": "image/png"
        },
        {
            "src": "/tc-report/assets/android-chrome-512x512.png",
            "sizes": "512x512",
            "type": "image/png"
        }
    ],
    "theme_color": "#ffffff",
    "background_color": "#ffffff",
    "display": "standalone"
}
```

Reply:
Awesome thanks Matt, I'll add the changes in.

Reply:
Changes added [71db81d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5840/commits/71db81d1d7cd01fb4f64db1f73c037820b1579be)